src/com/company/Main.java

```diff
@@ -20,53 +21,55 @@ public static void main(String[] args) {
 
         isFileFormatSupported(file);
 
-        if(!file.exists() || !isFileFormatSupported(file)) {
+        if (!file.exists() || !isFileFormatSupported(file)) {
```

Comment:
isFileFormatSupported is called in line 22 as well?

Reply:
it was duplicated,
but now the duplicated code has been removed.